controllers/common/secret-utils.go

```diff
@@ -157,6 +157,11 @@ func CreateUniqueName(params ...string) string {
 	return fmt.Sprintf("%x", sha512.Sum512([]byte(genStr)))
 }
 
+// CreateShortUniqueName function creates a name of 40 chars using sha512 hex sum from the given parameters
+func CreateShortUniqueName(params ...string) string {
```

Comment:
Can you make this more specific, as we already have a general function?
Rather than accepting n params, can this be more specific like,

`func CreateUniqueSecretName(managedClusterName, storageClusterNamespace, storageClusterName string)`

Reply:
Done